src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_ExpressionStatement.cs

```diff
@@ -26,6 +26,23 @@ public override BoundNode VisitExpressionStatement(BoundExpressionStatement node
             }
         }
 
+        public override BoundNode VisitLocalDeconstructionDeclaration(BoundLocalDeconstructionDeclaration node)
```

Comment:
The reason I have it here is that it follows from `VisitExpressionStatement` above. Unfortunately, I didn't find a good way to factor the common logic.